keps/sig-node/3570-cpumanager/README.md

```diff
@@ -1,91 +1,140 @@
+
+<!--
+**Note:** This checklist is iterative and should be reviewed and updated every time this enhancement is being considered for a milestone.
+-->
+
+[kubernetes.io]: https://kubernetes.io/
+[kubernetes/enhancements]: https://git.k8s.io/enhancements
+[kubernetes/kubernetes]: https://git.k8s.io/kubernetes
+[kubernetes/website]: https://git.k8s.io/website
+
+## Summary
+
+The *CPU Manager* is a new software component in Kubelet responsible for
+assigning pod containers to sets of CPUs on the local node. In later
+phases, the scope will expand to include caches, a critical shared
+processor resource.
+
+The kuberuntime notifies the CPU manager when containers come and
+go. The first such notification occurs in between the container runtime
+interface calls to create and start the container. The second notification
+occurs after the container is stopped by the container runtime. The CPU
+Manager writes CPU settings for containers using a new CRI method named
+[`UpdateContainerResources`](https://github.com/kubernetes/kubernetes/pull/46105).
+This new method is invoked from two places in the CPU manager: during each
+call to `AddContainer` and also periodically from a separate
+reconciliation loop.
 
-## Overview
+This KEP supersedes and replaces `kubernetes/enhancements/keps/sig-node/375-cpumanager/README.md`.
 
-_Problems to solve:_
+## Motivation
 
 1. Poor or unpredictable performance observed compared to virtual machine
    based orchestration systems. Application latency and lower CPU
    throughput compared to VMs due to cpu quota being fulfilled across all
    cores, rather than exclusive cores, which results in fewer context
    switches and higher cache affinity.
-1. Unacceptable latency attributed to the OS process scheduler, especially
+2. Unacceptable latency attributed to the OS process scheduler, especially
    for “fast” virtual network functions (want to approach line rate on
    modern server NICs.)
 
-_Solution requirements:_
+### Goals
 
 1. Provide an API-driven contract from the system to a user: "if you are a
    Guaranteed pod with 1 or more cores of cpu, the system will try to make
    sure that the pod gets its cpu quota primarily from reserved core(s),
    resulting in fewer context switches and higher cache affinity".
-1. Support the case where in a given pod, one container is latency-critical
+2. Support the case where in a given pod, one container is latency-critical
    and another is not (e.g. auxiliary side-car containers responsible for
    log forwarding, metrics collection and the like.)
-1. Do not cap CPU quota for guaranteed containers that are granted
+3. Do not cap CPU quota for guaranteed containers that are granted
    exclusive cores, since that would be antithetical to (1) above.
-1. Take physical processor topology into account in the CPU affinity policy.
-
-### Related issues
+4. Take physical processor topology into account in the CPU affinity policy.
```

Comment:
A NIT which was pointed out by SIG docs in one of my PRs: for ordered list we should use `1` for all the items as the markdown parser captures the correct number. This is beneficial if items are added or removed requiring no updates to the numbering.

Reply:
TIL! thanks, fixing (all across the document)